sdks/java/io/hadoop/jdk1.8-tests/src/test/resources/kubernetes/cassandra/LargeProductionCluster/start-up.sh

```diff
@@ -14,7 +14,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-#!/bin/sh
+#!/bin/bash -e
```

Comment:
I don't think this does what we want it to do.  (stop the script when it encounters an error)

I wasn't I tested this out on my machine, and it keeps on running past the line with the error. 
 adding the command "set -e" as the first line after this will do that.